requirements.txt

```diff
@@ -1,8 +1,10 @@
 kubernetes
 robotframework>=3.2.2
+backoff
```

Comment:
Still the version requirements are not reflected here
```
        'robotframework>=3.2.2',
        'backoff>=1.11.1',
        'urllib3-mock>=0.3.3'
```

Reply:
My bad, I'll fix that